packages/common/src/intl/locales/ar.json

```diff
@@ -51,9 +53,9 @@
   "label.color": "Colour",
   "label.comment": "Comment",
   "label.confirmed": "Confirmed",
+  "label.cost": "Cost",
```

Comment:
How to make a diff bigger: step1) update a bunch of stories with unneeded work. step 2) Alphabetically sort some lookup tables. Step3 ) make sure you do both in the same pr

Reply:
Guilty of doing both (1) and (2).. and (3) as well, now that I think of it.